feat: updating collection editor to use sub entity objects (#48)
BrightspaceHypermediaComponents/foundation-components: creating-new-components.md

Types are based on Siren's hypermedia format.

- `classes`: classes on the entity

> - `entity`: a dump of the entire entity object associated with this href

Reviewer: `a dump` what it means here?

Author: like a `var_dump` type thing - I can adjust the language

> - `entity`: a dump of the entire entity object associated with this href

Author: ```suggestion
- `entity`: the entire parsed entity object associated with this href
```
